protocols/v2/subprotocols/common-messages/src/lib.rs

```diff
@@ -1,7 +1,16 @@
+//! # Stratum V2 Common Messages Crate.
+//!
+//! This crate defines a set of shared messages used across all Stratum V2 subprotocols.
+//!
+//! ## Build Options
+//! This crate can be built with the following features:
+//! - `std`: Enables support for standard library features.
+//! - `serde`: Enables support for serialization and deserialization using Serde.
```

Comment:
```suggestion
//! - `serde`: Enables support for serialization and deserialization using Serde. Note that this feature flag is only used for the Message Generator, and deprecated for any other kind of usage. It will likely be fully deprecated in the future.
```

Reply:
This one is needed, otherwise it won't be rendered on crate docs